lighthouse-core/formatters/table.js

```diff
@@ -78,13 +78,18 @@ class Table extends Formatter {
 
     const rows = results.map(result => {
       const cols = headingKeys.map(key => {
+        let value = result[key];
+        if (typeof value === 'undefined') {
```

Comment:
https://github.com/GoogleChrome/lighthouse/pull/1536/files#diff-508b9167df0e380cb39ba29ab7deafa2R98 for example. I could use that over in the flexbox audit to fix the whitespace formatting of the code snippet.